src/Fleece.SystemJson/Fleece.SystemJson.fs

```diff
@@ -97,9 +97,9 @@ type [<Struct>] Encoding = Encoding of JsonValue with
 
     static member jsonOfJsonObject (o: JsonObject) = o :> JsonValue
 
-    static member createTuple c t = function 
-        | JArray a as x -> if length a <> c then Decode.Fail.count c (Encoding x) else t a
-        | a -> Decode.Fail.arrExpected (Encoding a)
+    // static member createTuple c t = function 
```

Comment:
Remove?

Reply:
Yes, thanks for the review. This is still WIP. I should have created it as a draft but instead I stated WIP in the description.